notebook2/notebook/templates/user.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" {{ 'class=dark' if session['dark'] }}>
+    <head>
+        <title>Hail User Profile</title>
+        <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"/>
+        <link rel="stylesheet" type="text/css" href="/static/css/main.css">
+        <link rel="stylesheet" type="text/css" href="/static/css/components/header.css">
+        <link rel="stylesheet" type="text/css" href="/static/css/pages/user.css">
+    </head>
+    <body id='theme-site'>
+        {% include 'header.html' %}
+        <div id="main">
+            <div class='centered'>
+                <div id='profile-header'>
+                    <img height="38" src={{user['picture']}} />
+                    <h3>{{user['name']}}</h3>
+                </div>
+                <div id='profile-data'>
+                    <h5>Google Data</h5>
+                    <p><b>Bucket Name: </b>{{user['bucket_name']}}</span></p>
+                    <p><b>Service Account: </b>{{user['gsa_email']}}</p>
```

Comment:
For this sort of thing I tend to use a definition list https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl, but I don't think it matters too much except for users with screen readers.